fish_probes/C_predict_probes.py

```diff
@@ -115,6 +161,22 @@ def check_uniqueness(kmers_precision, seq_other, probe_len):
         UTIL_log.print_message("The selected probes map to other "+str(n_matching_to_other)+" sequences.\n")
     return other_sel_clades
 
+def check_uniqueness_fast(kmers_precision, seq_other, probe_len):
+    n_matching_to_other = 0
+    # we check if the kmers are covered by other sequences
+    other_sel_clades = dict()
+    if len(kmers_precision.keys()) != 1:
```

Comment:
```suggestion
    if len(kmers_precision) != 1:
```
len(dict) == len(dict.keys())